master/docs/manual/cfg-www.rst

```diff
@@ -541,3 +545,23 @@ More complex config with separation per branch:
         ]
     )
     c['www']['authz'] = authz
+
+Using GitHub authentication and allowing access to all endpoints for users in the "BuildBot" organisation, using the avatars from GitHub:
+
+.. code-block:: python
+
+    from buildbot.plugins import *
+    from buildbot.www.avatar import AvatarGitHub
```

Comment:
since you imported `buildbot.plugins` after your change in `setup.py` (thank you!) you can use `util.AvatarGitHub` without extra import.